.github/workflows/cd.yml

```
@@ -27,6 +27,7 @@ env:
   AWS_REGION: ${{ secrets.AWS_REGION }}
   AWS_S3_PROFILE_IMAGE_PREFIX: ${{ secrets.AWS_S3_PROFILE_IMAGE_PREFIX }}
   AWS_S3_PUBLIC_BASE_URL: ${{ secrets.AWS_S3_PUBLIC_BASE_URL }}
+  OPENAI_API_KEY: {{ secrets.OPENAI_API_KEY }}
```

actionlint (1.7.12)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)